node/node-tests.ts

```diff
@@ -1998,6 +1999,31 @@ namespace repl_tests {
     }
 }
 
+///////////////////////////////////////////////////
+/// DNS Tests : https://nodejs.org/api/dns.html ///
+///////////////////////////////////////////////////
+
+namespace dns_tests {
+    dns.lookup("nodejs.org", (err, address, family) => {});
```

Comment:
you could add type specifiers for the callback parameters to allow typescript to check that they match (e.a. `(err: NodeJS.ErrnoException, address: string, family: number) => {}`)